risc0/zkvm/src/host/server/prove/tests.rs

```diff
@@ -572,8 +572,8 @@ mod sys_verify {
             .is_err());
 
         // TODO(#982) With conditional receipts, implement the following cases.
-        // verify with proven corraboration in verifier success.
-        // verify with unresolved corraboration in verifier success.
+        // verify with proven corroboration in verifier success.
+        // verify with unresolved corroboration in verifier success.
```

Comment:
I am aware that RISC Zero has specifically picked the word "corroboration" to mean evidence.